pkg/util/mount/nsenter_mount.go

```diff
@@ -175,8 +175,10 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 		glog.V(5).Infof("findmnt: directory %s does not exist", file)
 		return true, err
 	}
-
-	args := []string{"--mount=/rootfs/proc/1/ns/mnt", "--", n.absHostPath("findmnt"), "-o", "target", "--noheadings", "--target", file}
+	// Add --first-only option in case multiple mount points exist.
```

Comment:
I do think this would get a little clear with a comment like `// since we are testing for the absense of a mountpoint, it is sufficient to get only the first of multiple possible mountpoints using --first-only.